screenpy/actions/see_all_of.py

```diff
@@ -37,26 +40,30 @@ class SeeAllOf:
         )
```

Comment:
Should we add an example with only one test, to show it's OK? Or remove one of the tests from the second example?